fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalEntityTypeTags.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.convention.v2;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.convention.v2.TagRegistration;
+
+/**
+ * See {@link net.minecraft.registry.tag.EntityTypeTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public final class ConventionalEntityTypeTags {
+	private ConventionalEntityTypeTags() {
+	}
+
+	/**
+	 * Tag containing entity types that display a boss health bar.
+	 */
+	public static final TagKey<EntityType<?>> BOSSES = register("bosses");
+	public static final TagKey<EntityType<?>> MINECARTS = register("minecarts");
+	public static final TagKey<EntityType<?>> BOATS = register("boats");
+
+	/**
+	 * Entities should be included in this tag if they are not allowed to be picked up by items or grabbed in a way
+	 * that a player can easily move the entity to anywhere they want. Ideal for special entities that should not
+	 * be able to be put into a mob jar for example.
+	 */
+	public static final TagKey<EntityType<?>> CAPTURING_NOT_SUPPORTED = register("capturing_not_supported");
```

Comment:
This logic should not be controlled via tags. When this API was first designed, it was understood that Fabric Convention Tags were not designed to be used for modifying logic (at least directly); they were used for loot tables, recipes, etc.

Even if we were to reject that notion, this is still not fit for the tag system. Mods might use this as a security feature (instead of the proper safeguards), and data packs can override this inadvertently.

This is better fit, either as an interface injected API or a custom EntityType property.

Reply:
We added a tag for unmovable blocks in the convention tag API. This is a similar situation here.

Reply:
In addition, there's over like 20 mods that do capturing with most using their own entity type tags
https://imgur.com/svORGdt

So the idea is make a single tag that modders with uncapturable entities can add to and the these other mods can read from that single spot. Currently now, I have to hunt down every single capturing mod to add my entity to which now caused my data folder to have a ton of other mod's namespace folder just to add to their blacklist tags. It's really not ideal
https://github.com/TelepathicGrunt/Bumblezone/tree/1.20-Arch/common/src/main/resources/data

Fabric API will provide no functionality for this tag. It's just a standardized tag to let mods mark their entity as not capturable so that other mods can respect that tag if they wish to